.tutone.yml

```diff
@@ -336,14 +336,17 @@ packages:
           - name: entities
             max_query_field_depth: 2
             exclude_fields:
+              - relationships
               - slug
           - name: entity
             max_query_field_depth: 2
             exclude_fields:
+              - relationships
               - slug
           - name: entitySearch
             max_query_field_depth: 2
             exclude_fields:
+              - relationships
```

Comment:
This seems to just be removing this from the query. Should this also be removing the field(s) on the associated structs?

Reply:
Oh interesting... I'm not sure why tutone is still adding it to the structs on generation.  Removing from the query resolves the issue, and I guess would not be an API change if we left the field, they would just always be empty 🤷🏻‍♂️